VMware.CloudFoundation.PasswordManagement.psm1

```diff
@@ -8989,6 +8995,136 @@ Function Request-PasswordRotationPolicy {
 }
 Export-ModuleMember -Function Request-PasswordRotationPolicy
 
+Function Update-PasswordRotationPolicy {
+    <#
+        .SYNOPSIS
+        Updates the credential password rotation settings for a credential managed by SDDC Manager.
+
+        .DESCRIPTION
+        The Update-PasswordRotationPolicy cmdlet updates the credential password rotation settings for a credential managed
+        by SDDC Manager.
+        The cmdlet connects to the SDDC Manager using the -server, -user, and -pass values:
+        - Validates that network connectivity and authentication is possible to SDDC Manager.
+        - Updates the credential password rotation settings based on the credential criteria specified.
+
+        .EXAMPLE
+        Update-PasswordRotationPolicy -server sfo-vcf01.sfo.rainpole.io -user administrator@vsphere.local -pass VMw@re1! -domain sfo-m01 -resource vcenterServer -resourceName sfo-m01-vc01.sfo.rainpole.io -credential SSH -credentialName root -autoRotate disabled
+        This example disables the credential password rotation settings for a credential managed by SDDC Manager.
+
+        .EXAMPLE
+        Update-PasswordRotationPolicy -server sfo-vcf01.sfo.rainpole.io -user administrator@vsphere.local -pass VMw@re1! -domain sfo-m01 -resource vcenterServer -resourceName sfo-m01-vc01.sfo.rainpole.io -credential SSH -credentialName root -autoRotate enabled -frequencyInDays 90
+        This example enables the credential password rotation settings for a credential managed by SDDC Manager.
+
+        .PARAMETER server
+        The fully qualified domain name of the SDDC Manager instance.
+
+        .PARAMETER user
+        The username to authenticate to the SDDC Manager instance.
+
+        .PARAMETER pass
+        The password to authenticate to the SDDC Manager instance.
+
+        .PARAMETER domain
+        The name of the workload domain to retrieve the credential password rotation settings for.
+
+        .PARAMETER resource
+        The resource type to retrieve the credential password rotation settings for. One of: sso, vcenterServer, nsxManager, nsxEdge, ariaLifecycle, ariaOperations, ariaOperationsLogs, ariaAutomation, workspaceOneAccess, backup.
+
+        .PARAMETER resourceName
+        The name of the resource to retrieve the credential password rotation settings for.
+
+        .PARAMETER credential
+        The credential type to retrieve the user password rotation settings for. One of: ssh, api, audit, sso.
+
+        .PARAMETER credentialName
+        The name of the credential to retrieve the user password rotation settings for.
+
+        .PARAMETER autoRotate
+        Enable or disable the credential password rotation for the credential by SDDC Manager. One of: enabled, disabled.
+
+        .PARAMETER frequencyInDays
+        The number of days of warning before credential's password will be automatically rotated by SDDC Manager.
+        #>
```

Comment:
Does this have an extra tab?

Reply:
Amended in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/commit/8c963dffbbdd3e5938dc17bb3651b811f8963c40. 😄